android/jni/com/mapswithme/platform/SocketImpl.cpp

```diff
@@ -0,0 +1,35 @@
+#include "../core/jni_helper.hpp"
+#include "base/logging.hpp"
+#include "patform/socket.hpp"
```

Comment:
base should be after platform (or, in other words, platform before base). As I commented in some previous your requests, we follow inverse dependency order in inclusions list. Platform depends on base, so, in includes list, files from platform should be first.